DigitalLearningSolutions.Web/Views/SsoLinkage/Index.cshtml

```diff
@@ -0,0 +1,29 @@
+﻿@{
+  ViewData["Title"] = "Single Sign On Linkage";

```

Comment:
"Link your accounts" might be better. Also the URL for this page is currently "SsoLinkage" - "LinkAccount" would be preferable, please.

Reply:
This has now been modified